src/main/scala/org/nbastats/model/HasBasicStats.scala

```diff
@@ -1,10 +1,24 @@
 package org.nbastats.model
 
+import java.sql.Time
+
 trait HasBasicStats {
 
-  val games: Int
-  val pointsPerGame: Double
-  val reboundsPerGame: Double
-  val assistsPerGame: Double
+  val game: Int
```

Comment:
is `game` the index of the game? or the number of games played?

Reply:
it should be renamed gamesPlayed as it represents the number of games played